### ISSUE TITLE
xlr 5.0.0 installation error

### ISSUE THREAD

**Issue by ltutar** [xebialabs/puppet-xlrelease#26]
when installing XLR version 5.0.0, I get the following error:

```
otice: /Stage[main]/Xlrelease::Config/File[/opt/xl-release/xl-release-server/plugins/xlr-xlrelease-plugin-1.7.1.jar]/ensure: defined content as '{md5}2c309e9bf716f5d77fa61c71d15c8555'
Info: Class[Xlrelease::Config]: Scheduling refresh of Class[Xlrelease::Service]
Info: Class[Xlrelease::Service]: Scheduling refresh of Service[xl-release]
Error: Could not start Service[xl-release]: Execution of '/sbin/service xl-release start' returned 1: Starting xlrelease: see xlrelease log for more info[FAILED]
Error: /Stage[main]/Xlrelease::Service/Service[xl-release]/ensure: change from stopped to running failed: Could not start Service[xl-release]: Execution of '/sbin/service xl-release start' returned 1: Starting xlrelease: see xlrelease log for more info[FAILED]
Notice: /Stage[main]/Xlrelease::Service/Service[xl-release]: Triggered 'refresh' from 1 events
Notice: /Stage[main]/Xlrelease::Service/Xlrelease_check_connection[default]: Dependency Service[xl-release] has failures: true
Warning: /Stage[main]/Xlrelease::Service/Xlrelease_check_connection[default]: Skipping because of failed dependencies
Info: Class[Xlrelease::Service]: Unscheduling all events on Class[Xlrelease::Service]
```

XLR 5.0.0 introduces yajsw just like XLD. I will investigate it further. But already issuing the error.

**Comment by ltutar** [comment]
We need a version comparison like 

```

  if versioncmp($version , '4.9.99') < 0 {
    Xldeploy_setup['default'] ->
    file { "/etc/init.d/${productname}":
      content => template("xldeploy/xldeploy-initd-${::osfamily}.erb"),
      owner   => 'root',
      group   => 'root',
      mode    => '0700'
    }
  } else {
    Xldeploy_setup['default'] ->
    ini_setting { 'forkhack':
      path    => "${server_home_dir}/conf/xld-wrapper-linux.conf",
      setting => 'wrapper.fork_hack',
      value   => 'true',
    } ->
    exec {"/bin/echo ${os_user}|${server_home_dir}/bin/install-service.sh":
      unless => "/usr/bin/test -f /etc/init.d/${productname}",
    }
  }
```

See https://github.com/xebialabs-community/puppet-xldeploy/blob/master/manifests/server/config.pp for more information.

**Comment by superseb** [comment]
fixed by #29
